joomla/joomla-cms
# [4.0] Move system alerts inline

Pull Request for Issue #28810

### Summary of Changes
Moves system alerts inline.

Styling is reverted to default which is what is been used for other inline alerts (eg. sample data plugin). If styling changes are required, this is best done upstream on the custom elements repo.

![image](https://user-images.githubusercontent.com/2803503/80354758-11d2da80-886f-11ea-95ff-817270c9b385.png)

### Testing Instructions
Apply this patch and run `node build.js --compile-css` to update the changed SCSS. Check alerts are displayed correctly.


### Expected result



### Actual result



### Documentation Changes Required
